nemo/collections/nlp/models/language_modeling/megatron_gpt_prompt_learning_model.py

```diff
@@ -424,13 +440,13 @@ def on_validation_epoch_start(self) -> None:
         self._reconfigure_batch_sizes(gbs, mbs)
         return super().on_validation_epoch_start()
 
-    def validation_epoch_end(self, outputs):
-        if len(outputs) == 0:
+    def on_validation_epoch_end(self):
+        if len(self.validation_step_outputs) == 0:
```

Comment:
Nit: `self.validation_step_outputs` is initialized to `[]` ? So can we just do 
```
if not self.validation_step_outputs:
```

Reply:
Fixed.